### DISCUSSION TITLE
Re-train ESM-3 model

### DISCUSSION THREAD

**Discussion by anonimoustt** [evolutionaryscale/esm#98] [Q&A]
Hi,

Is it possible to fine-tune or re-train the ESM-3 model with the protein sequence and structure or pdb file, and then apply the fine-tuned model to generate sequence or structure embeddings?

**Comment by Mainframework** [comment]
OFC no answer, no one really understand how to benchmark really this weights **with the structure**.  I would like to compare it but how?
